tests/a010_ensure_python_version.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```

Comment:
When I run `pytest tests` from root directory this test isn't executed. Do you have the same issue?
```
mikel@msi:~/repo/DUNE$ pytest tests/                                                                                                    
========================================================= test session starts ==========================================================
platform linux -- Python 3.8.10, pytest-7.2.1, pluggy-0.13.0                                                                            
rootdir: /home/mikel/repo/DUNE                                                                                                          
collected 31 items                                                                                                                     

tests/a100_ensure_container_image_test.py .                                                                                      [  3%]
tests/test_all_options_tested.py s                                                                                               [  6%]
tests/test_antler_proj.py s                                                                                                      [  9%]
tests/test_bootstrap.py .                                                                                                        [ 12%]
tests/test_bootstrap_full.py .                                                                                                   [ 16%]
tests/test_container.py .                                                                                                        [ 19%]
tests/test_debug.py .                                                                                                            [ 22%]
tests/test_deploy.py .                                                                                                           [ 25%]
tests/test_help.py ..                                                                                                            [ 32%]
tests/test_keys.py .                                                                                                             [ 35%]
tests/test_list_features.py .                                                                                                    [ 38%]
tests/test_monitor.py .                                                                                                          [ 41%]
tests/test_nodes.py ..                                                                                                           [ 48%]
tests/test_plugin.py ..                                                                                                          [ 54%]
tests/test_project.py ....                                                                                                       [ 67%]
tests/test_remainder.py ..                                                                                                       [ 74%]
tests/test_version.py .                                                                                                          [ 77%]
tests/test_version_selector.py .....                                                                                             [ 93%]
tests/test_wallet.py ..                                                                                                          [100%]

============================================== 29 passed, 2 skipped in 300.12s (0:05:00) ===============================================
```

Reply:
Whoops! Fixed in 229ab2c.

The test must follow the format: `test_xxxx.py` or `xxxx_test.py`. At least for an automatic run, anyway.